plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events-v2.ts

```diff
@@ -387,7 +387,12 @@ export function addHistoricalEventsExportCapabilityV2(
     }
 
     async function exportHistoricalEvents(payload: ExportHistoricalEventsJobPayload): Promise<void> {
-        status.info('ℹ️', `Running export historical events with config ${pluginConfig} for payload: ${payload}`)
+        status.info(
+            'ℹ️',
+            `Running export historical events with config ${pluginConfig.id} for payload: ${JSON.stringify(
+                payload
+            )}`
+        )
```

Comment:
Fwiw we are using pino logger under the hood, which supports adding a context dict:

```suggestion
        status.info(
            'ℹ️',
            'Running export historical events', {
                pluginConfigId: pluginConfig.id,
                payload
            })
        )
```

Reply:
IMO this looks much cleaner, so I'll try to make it work.